clang/lib/AST/Interp/FunctionPointer.h

```diff
@@ -22,7 +22,11 @@ class FunctionPointer final {
   const Function *Func;
 
 public:
-  FunctionPointer() : Func(nullptr) {}
+  // FIXME: We might want to track the fact that the Function pointer
+  // has been created from an integer and is most likely garbage anyway.
+  FunctionPointer(int IntVal = 0, const Descriptor *Desc = nullptr)
```

Comment:
@tbaederr  This is causing a lot of MSVC warnings - `int IntVal` -> `intptr_t IntVal`?

`warning C4312: 'reinterpret_cast': conversion from 'int' to 'const clang::interp::Function *' of greater size`

Reply:
Sorry, will fix that.

Reply:
Ah sorry, looks like I'm too slow :upside_down_face:

Reply:
For reference @AaronBallman fixed this in 4d80dff819d1164775d0d55fc68bffedb90ba53c